js/analytics.js

```diff
@@ -5,15 +5,26 @@
 var URI = require('urijs');
 var _ = require('underscore');
 
+function trackerExists() {
+  if (typeof ga !== 'undefined') {
+    return true;
+  }
+}
```

Comment:
Ah that makes sense. Thanks for clarifying. I didn't set up the check for `ga` originally, but as far as I can tell it was there because external code was calling methods so that code had to check first.